### ISSUE TITLE
Headers overriding

### ISSUE THREAD

**Issue by AILazerka** [hboylan/ng2-http#30]
Hello!

When implementing service for backend requests I've found out that it'll be great to have feature when you're adding Headers decorator to the method with an opportunity to override headers from DefaultHeaders which are on class. Mostly I'm using application/json as the Content-Type but for several methods I want to use another. Currently ng2-http is appending existing comma-separated list of values in one string.

p.s. it's a suggestion for improvement

**Comment by hboylan** [comment]
Good catch. The intended use case is for local headers to override the defaults. If that’s not happening already this is definitely a bug

**Comment by AILazerka** [comment]
As far as I understood from sources only appending is applied but no replacing. But can be wrong. 
Doesn't work anyways))

**Comment by hboylan** [comment]
Thanks, I’ll investigate

**Comment by AILazerka** [comment]
Thank you very much!!!
